src/types/pointer_gestures_v1.zig

```diff
@@ -0,0 +1,77 @@
+const wlr = @import("../wlroots.zig");
+
+const std = @import("std");
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+const zwp = wayland.server.zwp;
+
+pub const PointerGesturesV1 = extern struct {
+    global: *wl.Global,
+    swipes: wl.list.Head(wl.Resource, null),
```

Comment:
Please double check this. It should be right, but I'm non quite sure.

Reply:
What you have here matches the wlroots ABI perfectly, so that's fine. We could also use `zwp.PointerGestureSwipeV1` here instead of `wl.Resource` which would be more specific.